src/isNumber.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+const isNumber = (val) => typeof val === 'number'
```

Comment:
type of `NaN` is number. Do we want to consider that a valid number for our use cases?

Reply:
hmm, if we're replicating lodash then I suggest we copy the code over here. At least the test suit so we know for sure we won't break anything.

Reply:
Yep, the idea was to use this instead lodash... but maybe I should delete this method and use directly Number.isFinite in the merchantMap as you said NaN shouldn't be a valid value...